spec/views/layouts/application.html.erb_spec.rb

```diff
@@ -66,13 +66,24 @@
   end
 
   context '<title>' do
-    it 'does not double-escape HTML in the title tag' do
-      view.title("Something with 'single quotes'")
+    context 'with a page title added' do
+      it 'does not double-escape HTML in the title tag' do
```

Comment:
maybe we could also add a spec for things like unescaped `>` to make sure that it does escape them correctly?